Client/Auth/index.ts

```diff
@@ -1,50 +1,39 @@
 import * as model from "../../model"
-import { PaxpayFeature } from "../../model/PaxpayFeature"
 import { Connection } from "../Connection"
+import { Session } from "./Session"
 
 export class Auth {
 	#roles?: string[]
-	#features?: PaxpayFeature[]
+	#features?: model.PaxpayFeature[]
 	#data: Partial<model.LoginResponse>
```

Comment:
if this is in the session, why leave it duplicated here?

Reply:
right